pipelines/skylab/smartseq2_multisample/Loom_schema.md

```diff
@@ -11,7 +11,7 @@ The global attributes in the Loom apply to the whole file, not any specific part
 | :-- | :-- |
 | LOOM_SPEC_VERSION | String with the loom file spec version |
 | CreationDate | Date Loom file was generated |
-| sample_id | The plateid listed in the pipeline configuration file; by default it is set to the batch_id. |
+| sample_id | The plateid listed in the pipeline configuration file; by default it is set to the batch_id |
```

Comment:
Does the global attribute still say sample_id on the Multi Sample SS2?

Reply:
It is batch_id and batch_name now

Reply:
```suggestion
| sequencing_input_id | The plate_id listed in the pipeline configuration file; by default it is set to the batch_id |
```